desktop/libs/notebook/src/notebook/templates/editor2.mako

```diff
@@ -891,7 +891,7 @@ There is no bridge to KO for components using this integration. Example using in
           <div class="editor-top-right-actions">
             <!-- ko template: { name: 'snippet-header-database-selection' } --><!-- /ko -->
             <button title="${ _('Expand editor') }" data-bind="toggle: $root.topExpanded">
-              <i class="fa" data-bind="css: { 'fa-expand': !$root.topExpanded(), 'fa-compress': $root.topExpanded() }"></i>
+              <i data-hue-analytics="editor:expand-editor" class="fa" data-bind="css: { 'fa-expand': !$root.topExpanded(), 'fa-compress': $root.topExpanded() }"></i>
```

Comment:
Nit: expand-editor-click for consistency?

Reply:
Yea, I had some idea that I would use "click" when the actual action was not completed by the interaction, e.g. a click on a save button that would prompt a confirmation dialog would be name "save-btn-click" and a click that actually did something all the way like expand a panel would be called "expand". But I'm not sure I managed to be consistent in that either....